themes/prism-gruvbox-dark.css

```diff
@@ -0,0 +1,154 @@
+/**
+ * Gruvbox dark theme
+ *
+ * Adapted from a theme based on:
+ * Vim Gruvbox dark Theme (https://github.com/morhetz/gruvbox)
+ *
+ * @author Azat S. <to@azat.io>
+ * @version 1.0
+ */
+
+:root {
+	--red: #fb4934;
```

Comment:
@mAAdhaTTah Do we allow CSS variables yet?

Reply:
Okay, I fixed it